charts/platform-service/templates/virtualservice.yaml

```diff
@@ -27,24 +34,33 @@ spec:
         host: {{ include "platform-service.fullQualifiedServiceName" . | quote }}
 {{- include "retries" . }}
 
-{{- if .Values.gateway.exposeService }}
-
 ---
+{{- if .Values.gateway.exposeService }}
 apiVersion: networking.istio.io/v1alpha3
 kind: VirtualService
 metadata:
   name: {{ include "platform-service.serviceName" . }}-external
+  labels:
+{{ include "appLabels" . | trim | indent 4 }}
 spec:
   hosts:
-{{- if .Values.defaultRouting.allHosts }}
-  - "*"
-{{- else }}
-  {{- range .Values.defaultRouting.hosts }}
-  - {{ . | quote }}
+  {{- if .Values.defaultRouting.kubeitDefaultHosts.enabled }}
+    - "{{ .Values.defaultRouting.kubeitDefaultHosts.hostsPrefix | default .Values.app }}{{ $tenantPrefix }}.{{ .Values.kubeit.clusterSubdomain }}.{{ .Values.defaultRouting.kubeitDefaultHosts.dnsDomain }}"                                                                      # Cluster specific subdomain, i.e.: dev001
+    - "{{ .Values.defaultRouting.kubeitDefaultHosts.hostsPrefix | default .Values.app }}{{ $tenantPrefix }}.{{ .Values.kubeit.shortRegion }}.{{ .Values.kubeit.environmentName }}.{{ .Values.defaultRouting.kubeitDefaultHosts.dnsDomain }}"                                      # F5/Active Cluster
+    - "{{ .Values.defaultRouting.kubeitDefaultHosts.hostsPrefix | default .Values.app }}{{ $tenantPrefix }}.{{ .Values.kubeit.clusterColour }}.{{ .Values.kubeit.shortRegion }}.{{ .Values.kubeit.environmentName }}.{{ .Values.defaultRouting.kubeitDefaultHosts.dnsDomain }}"   # Colour cluster - blue/green deployment
+  {{- if .Values.defaultRouting.kubeitDefaultHosts.tenantMultiRegion }}
+    - "{{ .Values.defaultRouting.kubeitDefaultHosts.hostsPrefix | default .Values.app }}{{ $tenantPrefix }}.{{ $.Values.kubeit.environmentName }}.{{ .Values.defaultRouting.kubeitDefaultHosts.dnsDomain }}"                                                                                                              # Traffic Manager
   {{- end }}
+  {{- end }}
+  {{- range .Values.defaultRouting.hosts }}
+    - {{ . | quote }}
 {{- end }}
   gateways:
-  - {{ required "A valid .Values.gateway.name entry required!" .Values.gateway.name }}
+{{- if or (.Values.defaultRouting.kubeitDefaultHosts.enabled) (eq .Values.kubeit.gatewayEnabled true) }}

```

Comment:
Is there any need to have two gateways defined at the same time? Seems like we should have only one...

Reply:
Verify where the values are defined, try passing the value as .values.gateway.name to the chart itself from the bootstrapping scripts

Reply:
Passing a value as `$.Values.gateway.name` from upstream manifest